evap/staff/importers/enrollment.py

```diff
@@ -594,12 +595,15 @@ def import_enrollments(
         if test_run:
             importer_log.add_success(_("The test run showed no errors. No data was imported yet."))
             msg = format_html(
-                _("The import run will create {evaluation_count} courses/evaluations and {user_count} users:{users}"),
-                evaluation_count=new_course_count,
-                user_count=len(new_user_profiles),
-                users=create_user_list_html_string_for_message(new_user_profiles),
+                _("The import run will create {evaluation_string} and {user_string}"),
+                evaluation_string=ngettext(
+                    "1 course/evaluation", "{count} courses/evaluations", new_course_count
+                ).format(count=str(new_course_count)),
```

Comment:
I don't think you need the explicit `str(...)`, the formatting should do it for you (also below)